### PR TITLE
Aidy/date timne add redux

### PR DESCRIPTION
**Pull request by aidy8847**
add datetime input and redux

### CODE REVIEW

File: client-side/src/App.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import './App.css';
 function App() {
   return (
     <div className="App">
         
-        <DateTime date={currentDate} format="full" />
-     <DateTime date={currentDate} format="date" />
-     <DateTime date={currentDate} format="time" />
+        <BasicDateTimeRangePicker/>
+     {/* <DateTime date={currentDate} format="date" />
```
Comment (ayeletBinder) [inline client-side/src/App.js:8]: תמחקי את ההערה

**Review by ayeletBinder — APPROVED** [review]

File: client-side/src/App.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+import BasicDateTimeRangePicker from './stories/DateTime'
 import './App.css';
 function App() {
   return (
     <div className="App">
         
-      
+        <BasicDateTimeRangePicker/>
+     {/* <DateTime date={currentDate} format="date" />
+     <DateTime date={currentDate} format="time" /> */}
```
Comment (SariGlick) [inline client-side/src/App.js:9]: remove datetime component from App.js

File: client-side/src/stories/DateTime.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { LocalizationProvider } from '@mui/x-date-pickers-pro/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers-pro/AdapterDayjs';
+import { DateTimeRangePicker } from '@mui/x-date-pickers-pro/DateTimeRangePicker';
```
Comment (SariGlick) [inline client-side/src/stories/DateTime.jsx:5]: put all imports from '@mui/x-date-pickers-pro' in one line
something like import {DemoContainer , LocalizationProvider , ...} from '@mui/x-date-pickers-pro'

File: client-side/src/stories/DateTime.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+
+
+
+export default function DateTime() {
+  return (
+    <div className='dateTime'>
+      <LocalizationProvider dateAdapter={AdapterDayjs} className='localizationProvider'>
+        <DemoContainer className="timeInput" components={['DateTimeRangePicker']}>
+          <DateTimeRangePicker localeText={{ start: 'start', end: 'end' }} className='DateTimeRangePicker'/>
```
Comment (SariGlick) [inline client-side/src/stories/DateTime.jsx:16]: get text from props, add default value of start & end

File: client-side/src/stories/DateTime.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.dateTime .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline {
+  border-color: rgb(72, 245, 193);
+}
+
+.dateTime .MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline {
+  border-color: rgb(72, 245, 193);
+}
+
+.dateTime .MuiOutlinedInput-root:hover .MuiOutlinedInput-notchedOutline {
+  border-color: rgb(72, 245, 193);
```
Comment (SariGlick) [inline client-side/src/stories/DateTime.scss:10]: put the color value in variable and use it for the 3 declarations

File: client-side/src/stories/DateTime.scss [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+.dateTime .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline {
+  border-width: 2px; 
+  border-style: solid; 
+  transition: border-color 0.3s ease; 
+}
+.dateTime{
+  width: 500px;
```
Comment (SariGlick) [inline client-side/src/stories/DateTime.scss:19]: dont use const pixels, use %

**Review by SariGlick — CHANGES_REQUESTED** [review]

**Pull request closed without merging**